charts/artifacthub-repo.yml

```diff
@@ -0,0 +1,6 @@
+repositoryID: a1115636-3f42-454c-8a76-a252a3f8dba3
```

Comment:
Where is this repository id?

Reply:
It gets created when the user publishes the repo on Artifact Hub.

https://artifacthub.io/docs/topics/repositories/#verified-publisher